apps/chat/src/components/Common/ApplicationDialog.tsx

```diff
@@ -212,18 +215,33 @@ const ApplicationDialogView: React.FC<Props> = ({
     }
 
     try {
-      const object: Record<string, string> = JSON.parse(data);
+      const object = JSON.parse(data);
+
+      if (typeof object === 'object' && object !== null) {
+        for (const [key, value] of Object.entries(object)) {
+          if (!key.trim()) {
+            return t('Keys should not be empty');
+          }
+
+          if (['boolean', 'number'].includes(typeof value) || value === null) {
+            continue;
+          }
 
-      for (const [key, value] of Object.entries(object)) {
-        if (!key.trim() || !value.trim()) {
-          return t('Keys and Values should not be empty');
+          if (typeof value === 'string' && !value.trim()) {
+            return t('String values should not be empty');
+          }
+
+          if (!['boolean', 'number', 'string', 'null'].includes(typeof value)) {
```

Comment:
you already checked boolean, number and null value. And `null` cannot be returned from `typeof`
```suggestion
          if (typeof value !== 'string') {
```